assets/src/users/UserContext.tsx

```diff
@@ -35,11 +40,11 @@ const AuthUserProvider: React.FC<{}> = ({ children }) => {
   );
 };
 
-const PayUserContext = React.createContext<User | undefined>(undefined);
+const PayUserContext = React.createContext<Partial<PayUserContext>>({});
 const UserContext = React.createContext<
   | {
       authUser: Auth0User;
-      payUser: User;
+      payUser: Partial<PayUserContext>;
```

Comment:
@acodegirl is there a specific behaviour you're going for, or is this mostly about making TS be quiet? I checked out the branch locally and is this a better solution for these two contexts?

```ts
const PayUserContext = React.createContext<PayUserContext>({
  isLoading: false
});
const UserContext = React.createContext<
  | {
      authUser: Auth0User;
    }
  | undefined
>(undefined);
```

Note: I removed `payUser` from the `UserContext`. It's not used anywhere, so I don't think it needs to be in there, at least not for now?